src/node_contextify.cc

```diff
@@ -647,6 +651,26 @@ class ContextifyScript : public BaseObject {
         True(env->isolate()));
   }
 
+  static bool GetBreakOnSigintArg(const FunctionCallbackInfo<Value>& args,
+                                  const int i) {
+    if (args[i]->IsUndefined() || args[i]->IsString()) {
+      return false;
+    }
+    if (!args[i]->IsObject()) {
+      Environment::ThrowTypeError(args.GetIsolate(),
+                                  "options must be an object");
+      return false;
+    }
+
+    Local<String> key = FIXED_ONE_BYTE_STRING(args.GetIsolate(),
+                                              "breakOnSigint");
+    Local<Value> value = args[i].As<Object>()->Get(key);
+    if (value->IsUndefined()) {
+      return false;
+    }
```

Comment:
Not strictly necessary, `value->BooleanValue() == false` when `value->IsUndefined() == true` (and you can shorten `return value->Booleanvalue()` to `return value->IsTrue()`.)